libcusp/include/galois/graphs/DistributedGraph.h

```diff
@@ -530,25 +530,38 @@ class DistGraph {
 
   std::vector<std::vector<size_t>>& getMirrorNodes() { return mirrorNodes; }
 
+private:
+  virtual unsigned get_host_id_impl(uint64_t) const = 0;
```

Comment:
Nitpick: I prefer camel case, especially since a majority of the names in this file are camel case.
Another thing I had been wanting to do for years is to make everything in libdist and friends camel case, but that would require quite a bit of refactoring.

Keeping it consistent where we can should be prioritized.

Reply:
I do agree with the style consistency! However, it's difficult to refactor as you said. I personally follow [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Variable_Names), but will change it to camel case as the original file adopts.

Reply:
Yes, at the moment the Galois codebase is pretty inconsistent with the function casing.

Reply:
As a side note, I vastly prefer camel casing (because underscores are very un-ergonomic: underscore key is typically located in a place that needs the pinky to hit, and making the pinky do more work is terrible on your hands since it's the weakest finger on your hand).

Style wise the underscores make vars easier to read, but the ergonomic tradeoff is bleh.

Reply:
I agree with Loc but ignore this change for now. It is fine as is. It's not a priority right now.